Framework/Muon/src/MuonPreProcess.cpp

```diff
@@ -214,18 +235,68 @@ MatrixWorkspace_sptr MuonPreProcess::applyTimeOffset(MatrixWorkspace_sptr ws,
   }
 }
 
+MatrixWorkspace_sptr
+MuonPreProcess::applyTimeZeroTable(MatrixWorkspace_sptr ws,
+                                   const TableWorkspace_sptr &timeZeroTable) {
+  auto cloneWs = cloneWorkspace(ws);
+  if (timeZeroTable != nullptr) {
```

Comment:
This is more a style nit, I personally like to remove a layer of indent where possible by doing

```
if (!timeZeroTable){
    return clone ws
}

// Note how this no longer is in an if block
const auto numSpec = cloneWs->getNumberHistograms();
....
return cloneWs
```

Secondly pointers are implicitly convertible to bool in the standard so
`if (foo != nullptr)` can be written as `if (foo)`

See: https://en.cppreference.com/w/cpp/memory/shared_ptr/operator_bool